addons/stock/models/stock_quant.py

```diff
@@ -104,6 +104,8 @@ def _domain_product_id(self):
     inventory_date = fields.Date(
         'Scheduled Date', compute='_compute_inventory_date', store=True, readonly=False,
         help="Next date the On Hand Quantity should be counted.")
+    inventory_quantity_set = fields.Boolean(default=False)  # Only used for UI purposes in the Inventory Adjustment page
```

Comment:
Boolean fields are already `False` by default
```suggestion
    inventory_quantity_set = fields.Boolean()  # Only used for UI purposes in the Inventory Adjustment page
```